index.js

```diff
@@ -52,15 +52,26 @@ function spawnClangFormat(args, done, stdio) {
   }
 
   // extract glob, if present
-  var filesGlob = args.filter(function(arg){return arg.indexOf('--glob=') === 0;})
+  var globArg = args.filter(function(arg){return arg.indexOf('--glob=') === 0;})
```

Comment:
Only the first one would be used, via the `.shift()` below. 

Maybe it's possible to support multiple globs, via the `+(pattern|pattern|pattern)` format described in the [primer](https://github.com/isaacs/node-glob#glob-primer), but I didn't try it yet.